From 76efa53fc4a9e0eb4fd03bdfc8845464d5250879 Mon Sep 17 00:00:00 2001
From: Olivier Desenfans <desenfans.olivier@gmail.com>
Date: Tue, 28 Jun 2022 15:24:49 +0200
Subject: [PATCH] Rename sevsecret to efisecret

Compatibility patch for Grub. The first version of James Bottomley's
patches for Grub named the secret loading module sevsecret.
This module was later renamed to efisecret as it is also compatible
with Intel TDX. This change was not picked up in EDK2.
---
 OvmfPkg/AmdSev/Grub/grub.cfg | 2 +-
 OvmfPkg/AmdSev/Grub/grub.sh  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/OvmfPkg/AmdSev/Grub/grub.cfg b/OvmfPkg/AmdSev/Grub/grub.cfg
index 17be94277a..fb1b6bba79 100644
--- a/OvmfPkg/AmdSev/Grub/grub.cfg
+++ b/OvmfPkg/AmdSev/Grub/grub.cfg
@@ -10,7 +10,7 @@
 ##
 
 echo "Entering grub config"
-sevsecret
+efisecret
 if [ $? -ne 0 ]; then
     echo "Failed to locate anything in the SEV secret area, prompting for password"
     cryptomount -a
diff --git a/OvmfPkg/AmdSev/Grub/grub.sh b/OvmfPkg/AmdSev/Grub/grub.sh
index 99807d7291..abec80d7da 100644
--- a/OvmfPkg/AmdSev/Grub/grub.sh
+++ b/OvmfPkg/AmdSev/Grub/grub.sh
@@ -44,7 +44,7 @@ GRUB_MODULES="
             linux
             linuxefi
             reboot
-            sevsecret
+            efisecret
             "
 basedir=$(dirname -- "$0")
 
-- 
2.25.1

